src/sign-in.js

```diff
@@ -2,11 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
-import { fetchCreateAccount, fetchSignIn } from './fetch-data';
+import { fetchSignIn, fetchUserObject } from './fetch-data';
 import { updateInitialState } from './reducer-handlers';
 
 let SignInDumb = ({ history, updateInitialState}) =>
     <div>
+        {
```

Comment:
This seems too good to be true.  I would have expected to use a class and lifecycle hook.  Must ask about potential downside of this solution.